Intro Skip/SkipDaemon.cs

```diff
@@ -53,6 +53,8 @@ public void Initialize()
             float firstObjectTime = _initData.audioClip.length;
             float lastObjectTime = -1f;
 
+            if (beatmapDataItems.Where(m => (m is NoteData || m is ObstacleData)).Count() == 0) return;
```

Comment:
This will allocate an array of an unknown twice (once for the .Where and another for the .Count). Do you think you could write this without LINQ?